epub33/core/index.html

```diff
@@ -181,23 +181,27 @@ <h3>Relationship to Other Specifications</h3>
 
 				<section id="sec-overview-relations-html">
 					<h4>Relationship to HTML</h4>
-					<p>This specification does not reference a specific version of W3C [[HTML]], but instead uses an
-						undated reference that will always point to the latest recommendation. This approach ensures
-						that EPUB will always keep pace with changes to the HTML standard. Authors will need to keep
-						track of changes to HTML, and ensure that their processes are kept up to date.</p>
 
-					<div class="caution">
+					<p>The [[HTML]] standard is continuously evolving &#8212; there are no longer versioned releases of
+						it. That standard, in turn, references various technologies that continue to evolve, such as
+						MathML, SVG, CSS and JavaScript.</p>
+
+					<p>The benefit of this approach for EPUB is that EPUB Publications always keep pace with changes to
+						the web without the need for new revisions. Authors, however, will need to keep track of the
+						various changes to HTML, and the technologies it references, and ensure that their processes are
```

Comment:
the web -> the Web

(Again the W3C house style requires a capitalization of the 'Web')